### PR TITLE
Fixes example to call things appropriately

### PR DESCRIPTION
**Pull request by skrawcz**
Fixes example to call things appropriately

### CODE REVIEW

File: examples/pandas/materialization/my_script.py [hunk 1]
```diff
@@ -41,24 +31,31 @@
...
-# let's create the dataframe!
-# df = dr.execute(output_columns)
 
 materializers = [
     # materialize the dataframe to a pickle file
     to.pickle(
         dependencies=output_columns,
         id="df_to_pickle",
-        path="./data/df.pkl",
-        combiner=adapter,  # add from hamilton import base
```
Comment (skrawcz) [inline examples/pandas/materialization/my_script.py:53]: sorry I had a typo -- it should be `combine` not `combiner`.

**Review by benhhack — APPROVED** [review]
Thanks for this! Hope it wasn't too much trouble.

**Comment by benhhack** [comment]
I merged the changes and wrote up the notebook, I just can't run the notebook because of a hamilton module not found error. I always get these in my Jupyter and am not of sure of the best fix, do you have any tips?

**Comment by skrawcz** [comment]
> I merged the changes and wrote up the notebook, I just can't run the notebook because of a hamilton module not found error. I always get these in my Jupyter and am not of sure of the best fix, do you have any tips?

what's the error? Did you pip install `sf-hamilton` ?

**Pull request merged**